ldms/man/ldmsd_sampler_discovery.man

```diff
@@ -0,0 +1,245 @@
+\" Manpage for ldmsd_sampler_discovery
+.TH man 7 "27 March 2024" "v5" "LDMSD Sampler Discovery man page"
+
+.\""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""/.
+.SH NAME
+ldmsd_sampler_disconvery - Manual for LDMSD Sampler Discovery
+
+.\""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""/.
+.SH SYNOPSIS
+
+**Sampler side Commands**
+
+.IP \fBadvertise_add
+.RI "name=" NAME " xprt=" XPRT " host=" HOST " port=" PORT
+.RI " [auth=" AUTH_DOMAIN "]"
+
+.IP \fBadvertise_start
+.RI "name=" NAME
+
+.IP \fBadvertise_stop
+.RI "name=" NAME
+
+.IP \fBadvertise_del
+.RI "name=" NAME
+
+.IP \fBadvertise_status
+.RI " [name=" NAME "]"
+
+.PP
+**Aggregator Side Commands**
+
+.IP \fBprdcr_listen-add
+.RI "name=" NAME " regex=" REGEX " " RECONNECT=" INTVL
```

Comment:
This needs to include a parameter to specify matching a specific user owning the producer (i.e. a user-owned sampler daemon on a remote host must be blockable from getting included on the basis of a munge confirmed failure to be uid 0 (if uid 0 is required). I expect there may be some scenarios where this is optional and an admin might want to allow user-owned daemons to publish.